tests/unit/test_tcp.py

```diff
@@ -1,6 +1,6 @@
 import struct
 
-from umodbus.client.tcp import _create_request_adu, _create_mbap_header
+from umodbus.client.tcp.tcp import _create_request_adu, _create_mbap_header
```

Comment:
Is this necessary even though there is a * import in the umodbus.client.tcp.__init__?

Reply:
Yes, the "*" only imports *visible* symbols. See [here](https://s16h.medium.com/importing-star-in-python-88fe9e8bd4d2) for a better explanation.
I was also surprised. I guess it is a python 3 feature. I don't remember python 2 behaving like this.

If I move everything to `__init__.py` this won't be an issue anymore. So +1 for moving the code to `__init__.py` :-)